app/controllers/registrations_controller.rb

```diff
@@ -3,7 +3,7 @@ class RegistrationsController < Devise::RegistrationsController
 
   def sign_up_params
     params.require(:user).permit(:first_name, :last_name, :email,
-                                 :password, :password_confirmation)
+                                 :password, :password_confirmation, :profile_photo)
```

Comment:
Line is too long. [83/80]<br>Align the parameters of a method call if they span more than one line.